libs/dao-data/src/Query.ts

```diff
@@ -83,8 +83,17 @@ import { createPaging, DEFAULT_RECORDS_PER_PAGE } from './utils';
 export default class Query {
   public endpoints: KeychainList;
 
-  constructor() {
+  constructor(graphApiKeys?: Keychain) {
     this.endpoints = ENDPOINTS;
+
+    if (graphApiKeys) {
```

Comment:
is this still pending?

Reply:
@santteegt whoops!

Yeah I better implement that lol